### DISCUSSION TITLE
Welcome to fountainpencompanion Discussions!

### DISCUSSION THREAD

**Discussion by ujh** [ujh/fountainpencompanion#2076] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TechScar** [comment]
Hi there!
I'm a professional software developer who fell into the fountain pens rabbit hole and came across this very neat open source project. Having pen and ink management like this is a fantastic thing, and I already see potential for a couple of features I'd like to implement 😄 This is my first venture into Ruby and React, so do please be forgiving of beginner's mistakes and review my PRs well - I wish to learn!

**Reply by ujh** [reply]
So great to have you on board!
